services/workflows-service/src/collection-flow/collection-flow.service.ts

```diff
@@ -324,4 +327,39 @@ export class CollectionFlowService {
       currentProjectId,
     );
   }
+
+  async uploadNewFile(projectId: string, workflowRuntimeDataId: string, file: Express.Multer.File) {
+    // upload file into a customer folder
+    const customer = await this.customerService.getByProjectId(projectId);
+
+    const runtimeDataId = await this.workflowService.getWorkflowRuntimeDataById(
+      workflowRuntimeDataId,
+      {},
+      [projectId],
+    );
+
+    const entityId = runtimeDataId.businessId || runtimeDataId.endUserId;
+
+    if (!entityId) {
+      throw new NotFoundException("Workflow does't exists");
+    }
+
+    // Remove file extension (get everything before the last dot)
+    const nameWithoutExtension = (file.originalname || generateRandomId(16)).replace(
```

Comment:
use uuid instead of random. simpler without replacing anything

Reply:
i prefer this method in order to control what symbols the generated string will include and also the length.
i can replace it with uuid if there concrete problem